<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addProcess</class>
 <widget class="QDialog" name="addProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#addProcess{
	background-image:url(img/addProcess.jpg);
}
QTextEdit{
	background-color:rgb(235,199,255);
}  
     
QLineEdit{
	background-color:rgb(235,199,255);
}  
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>进度名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="processName">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>进度描述：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnConfirm">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>210</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>210</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="Info">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;黑体&quot;;
color:rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="Describe">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="proName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>68</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>102</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>进度管理的任务名：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="processTaskName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
